gbmodel/model_sqlalchemy.py

```diff
@@ -432,18 +464,26 @@ def check_review_state(self, session_id, date):
 class reports(db.Model):
     __table__ = db.Model.metadata.tables['reports']
 
-    def get_reports_for_student(self, student_id, term_id, is_final=None):
+    def get_reports_for_student(self, student_id, session_id, is_final=None):
         """
-        Gets all available reports for a student, optionally filtering to only midterms or finals.
+        Gets all available reports for a student, optionally filtering to only midterms or finals
```

Comment:
I wonder if we might consider changing the word "reports" to "reviews" here. If so, it might be wise to go through the entire code base and do this kind of thing

Reply:
It'd be a good thing for code clean up, I don't think we want to try doing that now